### ISSUE TITLE
GitHub section of Home page is cut when large font is used

### ISSUE THREAD

**Issue by xperiandri** [github/VisualStudio#159]
With large font GitHub section of Home page is cut 

<img width="335" alt="github section height" src="https://cloud.githubusercontent.com/assets/2365592/11546149/22a971a2-9954-11e5-9160-417d9cd040d7.png">

Please, do not constraint its height, use appropriate panel for layout.

______________

You can debug this issue by installing Productivity Power tools.
Then:
* Invoke PresentEdit command
```
<?xml version="1.0"?>
<PresenterModeSettings>
  <EnvironmentFontFamily>Segoe UI</EnvironmentFontFamily>
  <EnvironmentFontSize>16</EnvironmentFontSize>
  <TextEditorFontFamily>Consolas</TextEditorFontFamily>
  <TextEditorFontSize>26</TextEditorFontSize>
</PresenterModeSettings>
```
* Save
* Invoke PresentOn command

**Comment by xperiandri** [comment]
Is there a build to try?

**Comment by shana** [comment]
@xperiandri This is included on the 1.0.18.4 release, out today!
